connectors/src/connectors/slack/index.ts

```diff
@@ -594,6 +594,26 @@ export async function retrieveSlackContentNodes(
   return new Ok(contentNodes);
 }
 
+export async function getWhiteListedChannelPattern(
+  connectorId: ModelId
+): Promise<Result<string | null, Error>> {
+  const slackConfig = await SlackConfigurationResource.fetchByConnectorId(
+    connectorId
+  );
+  if (!slackConfig) {
+    return new Err(
+      new Error(
+        `Failed to find a Slack configuration for connector ${connectorId}`
+      )
+    );
+  }
+  if (!slackConfig.whiteListedChannelPattern) {
```

Comment:
It can simply be Ok(slackConfig.whiteListedChannelPattern) instead it these 3 lines ?

Reply:
Wen need to handle when `lackConfig.whiteListedChannelPattern` is `undefined` otherwise the returned type needs to be changed to `Promise<Result<string | null | undefined, Error>>`.